models/order/src/return-info/transformers.ts

```diff
@@ -0,0 +1,19 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TReturnInfo, TReturnInfoGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TReturnInfo, TReturnInfo>('default', {
+    buildFields: [],
+  }),
+  rest: Transformer<TReturnInfo, TReturnInfo>('rest', {
+    buildFields: [],
+  }),
+  graphql: Transformer<TReturnInfo, TReturnInfoGraphql>('graphql', {
+    buildFields: [],
```

Comment:
I think `items` needs to be built. I assume this was the reason for the issue in the builder.spec file above where you left a comment

Reply:
[228273d](https://github.com/commercetools/test-data/pull/669/commits/228273d07306b97f0119f2a1645f3557186851ad) thank you! I also forgot to fake it inside the generator :see_no_evil: